cli/pkg/resourcemanager/client.go

```diff
@@ -15,8 +15,7 @@ type Client struct {
 	client             *HTTPClient
 	resourceName       string
 	resourceNamePlural string
-	deleteSuccessMsg   string
-	tableConfig        TableConfig
+	options            options
```

Comment:
3 options seemed enough to move them into its own struct, so it can grow in the future and still havig the `client` struct readable